### PR TITLE
Freewheel Doc

### PR DESCRIPTION
**Pull request by jitendraharpalani58**
@davisshaver review/merge ?

### CODE REVIEW

File: wordpress-development/ads.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+We are mostly using `tq.caid` (for ad targeting) and `tq.section` (for site section id). TK: What the other variables could be used for.
+
+####Known issues
+* We don't have access to Freewheel Admin as the contract lies with ABC News. Only ABC employees are allowed to access Freewheel Admin. This causes some bottleneck issues with producing new campaigns and diagnosing issues, so we hope to mitigate in the future.
+
+## Polar
+Polar is implemented for branded content. This integration consists of two Polar touch points:
+* [Media Voice](https://mediavoice.com/present) (Polar CMS): This is where ad sales team populates data
+* [Plugin to generate script](http://plugin.mediavoice.com/bookmarklet/ui/index.html#)
+Handlebar template: Polar uses handlebar js to generate the ad.
```
Comment (davisshaver) [inline wordpress-development/ads.md]: @jitendraharpalani58 hi! could you clarify this a bit? The integration consists of two touch points with polar – just want to confirm, we use the Plugin site by Polar to generate the handlebar template. Is that correct?

**Comment by davisshaver** [comment]
@jitendraharpalani58 this looks great. I learned a lot. One small question and then we should merge this.

File: wordpress-development/ads.md (hunk 1 above)
Comment (jitendraharpalani58) [inline wordpress-development/ads.md]: Yes, we use that plugin to generate handlebar template.

Comment (danielbachhuber) [inline wordpress-development/ads.md]: 44451a824205bfc82ca0ef2e88890dbf5c21770d

**Comment by danielbachhuber** [comment]
@jitendraharpalani58 :100:

**Pull request merged**